shell/platform/windows/system_utils.h

```diff
@@ -10,6 +10,11 @@
 #include <string>
 #include <vector>
 
+#include "flutter/shell/platform/windows/windows_registry.h"
+
+#define WIN_REG_I18N_PROFILE L"Control panel\\International\\User Profile"
+#define WIN_REG_LANGUAGES L"Languages"
```

Comment:
I'd be consistent with how we've defined other registry keys. For example:  https://github.com/flutter/flutter/blob/73e8c0233cc1a12ed2f70bc90343a294acdf38b6/packages/flutter_tools/templates/app_shared/windows.tmpl/runner/win32_window.cpp#L25-L27

Consider something like:

```suggestion
// Registry key for user-preferred languages.
constexpr const wchar_t kGetPreferredLanguageRegKey[] =
  L"Control panel\\International\\User Profile";
constexpr const wchar_t kGetPreferredLanguageRegValue[] = L"Languages";
```